.travis.yml

```diff
@@ -2,5 +2,9 @@
 language: python
 python:
     - "2.7"
-install: pip install -r files/default/test-requirements.txt
-script: python files/default/test-neutron-ha-tool.py
+install:
+    - pip install -r files/default/test-requirements.txt
+script:
+    - python files/default/test-neutron-ha-tool.py
+    - flake8 neutron-ha-tool.py test-neutron-ha-tool.py
```

Comment:
I would test first the flake8 and after that the tests, so first fail with the pep8 errors